lib/resorts/cypress-mountain/index.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  selector: '#conditions_lifts_1339bb0d2df6247a6d9778c66b41883b div.breakInsideAvoid',
```

Comment:
I simplified selector to `'[id^=conditions_lifts_] div.breakInsideAvoid'` - trying to avoid checking in magic numbers.